src/js/server/lib/password.js

```diff
@@ -0,0 +1,34 @@
+/*
+
+    A static function to consistently encode a password.  Used when storing passwords in CouchDB, and when logging in
+    using a username and password.
+
+    This package uses the `pbkdf2` functions provided by the `crypto` package built into node.  It has been tested to
```

Comment:
Personally I would have preferred bcrypt but it seems that pbkdf2 is a perfectly reasonable choice: 
http://security.stackexchange.com/questions/4781/do-any-security-experts-recommend-bcrypt-for-password-storage